polynote-frontend/polynote/ui/component/notebook/cell.ts

```diff
@@ -806,11 +806,9 @@ abstract class MonacoCell extends Cell {
             const model = this.editor.getModel();
             if (model) {
                 const range = new PosRange(model.getOffsetAt(evt.selection.getStartPosition()), model.getOffsetAt(evt.selection.getEndPosition()));
-                if (evt.selection.getDirection() === SelectionDirection.RTL) {
-                    this.cellState.updateField("currentSelection", () => range.reversed)
-                } else {
-                    this.cellState.updateField("currentSelection", () => range)
-                }
+                // Monaco guarantees that the position from Selection.getStartPosition() will be before or
+                // equal to Selection.getEndPosition(), so we don't need to worry about selection direction
+                this.cellState.updateField("currentSelection", () => range)
```

Comment:
Ah good find! So we were reversing the selection ourselves. I assume Monaco's behavior must have changed at some point in the past few years... 🤔

Reply:
Probably from when I updated Monaco a few months ago...oops 🤦‍♂️

Reply:
We are going to have to keep an eye on Monaco... whenever we update it, we should expect a lot of breakage.